Samples/word-citation-management/manifest.json

```diff
@@ -0,0 +1,215 @@
+        "full": "Citation manager - sample"
+    },
+    "description": {
+        "short": "A template to get started with citation management.",
+        "full": "A template to get started with citation management."
+    },
+    "icons": {
+        "outline": "assets/icon-32.png",
+        "color": "assets/icon-64.png"
+    },
+    "accentColor": "#230201",
+    "validDomains": [
+        "https://www.contoso.com",
+        "https://localhost:3000"
+    ],
+    "showLoadingIndicator": false,
+    "isFullScreen": false,
+    "defaultBlockUntilAdminAction": false,
+    "authorization": {
+        "permissions": {
+            "resourceSpecific": [
+                {
+                    "name": "Document.ReadWrite.User",
+                    "type": "Delegated"
+                }
+            ]
+        }
+    },
+    "extensions": [
+        {
+            "requirements": {
+                "scopes": [
+                    "document"
+                ]
+            },
+            "runtimes": [
+                {
+                    "requirements": {
+                        "capabilities": [
+                            {
+                                "name": "Taskpane",
+                                "minVersion": "1.1"
+                            }
+                        ],
+                        "scopes": [
+                            "document"
+                        ],
+                        "formFactors": [
+                            "desktop"
+                        ]
+                    },
+                    "id": "runtime_1",
+                    "type": "general",
+                    "code": {
+                        "page": "https://localhost:3000/taskpane.html",
+                        "script": "https://localhost:3000/taskpane.js"
+                    },
+                    "lifetime": "short"
+                },
+                {
+                    "requirements": {
+                        "capabilities": [
+                            {
+                                "name": "AddinCommands",
+                                "minVersion": "1.1"
+                            }
+                        ]
+                    },
+                    "id": "runtime_2",
+                    "type": "general",
+                    "code": {
+                        "page": "https://localhost:3000/taskpane.html",
+                        "script": "https://localhost:3000/taskpane.js"
+                    },
+                    "lifetime": "short",
+                    "actions": [
+                        {
+                            "id": "ButtonId1_1",
+                            "type": "openPage",
+                            "pinnable": false,
+                            "view": "ButtonId1",
+                            "multiselect": false,
+                            "supportsNoItemContext": false
+                        }
+                    ]
+                }
```

Comment:
Something's gone wrong here. The "capabilities" asks for AddinCommands 1.1. But the "actions.type" is "openPage" which is for task panes, not add-in commands. Also, the two URLs in "code" are for a task pane. There's no add-in command in this sample and you've already got a runtime for the task pane, just above this one, so I think this should just be deleted.

Reply:
> Actually, I thought this was for the ribbon button, so when you click on it, it opens the task pane...

Based on what I see in the original XML manifest, this add-in has only one button and that is a "Show Task Pane" button. But the JSON manifest has two runtime objects in the "runtimes" array. The first one looks correct to be the runtime for the task pane. But this second one seems to have no purpose, and it's an odd mixture of values that are appropriate for a Show Task Pane button and values that are appropriate for a function command button.